saifa/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
+
+services:
+  backend:
+    image: patranakamo/saifa-be:v1.3.3
+    restart: on-failure
```

Comment:
Perfect. You'll want to add this same `restart: on failure` to the frontend and redis services in this compose file as well.